core/src/main/scala/akka/persistence/r2dbc/internal/R2dbcExecutor.scala

```diff
@@ -79,8 +79,8 @@ import reactor.core.publisher.Mono
       val consumer: BiConsumer[mutable.Builder[A, IndexedSeq[A]], A] = (builder, elem) => builder += elem
       Flux
         .from[A](result.map((row, _) => mapRow(row)))
-        .collect(() => IndexedSeq.newBuilder[A], consumer)
-        .map(_.result())
+        .collect[scala.collection.mutable.Builder[A, IndexedSeq[A]]](() => immutable.IndexedSeq.newBuilder[A], consumer)
```

Comment:
not important but for implementation details it could be `Vector.newBuilder`

Reply:
I thought so as well and tried vector but couldn't get it to compile with the Java Flux/Mono method type params.

Reply:
Gave it another try but I'm giving up. Johan vs Scala 2.12 and Java lambda invariance: 0 - 1